views/templates/admin/configuration.tpl

```diff
@@ -22,3 +22,10 @@
 <div id="app"></div>
 
 <script src="{$pathApp|escape:'htmlall':'UTF-8'}"></script>
+
+<style>
+  /** Hide native multistore module activation panel */
```

Comment:
Please provide an explanation on the reason you hide this. Something like:

```suggestion
  /** Hide native multistore module activation panel because visual regressions on non-bootstrap content */
```